[DOC] add faq (#16)
bids-standard/bids-extensions: docs/schema.md

The schema source code lives within the [BIDS Specification][schema_source]; 
  a deferenced JSON'ified verison can be found at the following url ->
  [bids-specification.readthedocs.io/en/stable/schema.json][jsonified_schema].

## Schema documentation

> A full desctiption of the schema can be found on this [website][bidsschematools_rtd] 

Reviewer: ```suggestion
A full description of the schema can be found on this [website][bidsschematools_rtd] 
```

where you will also find the documentation for the python package

> to interact with the schema, [bidsschematools][bidsschematools_pypi].

Author: - [ ] cross check that this is not mentioned already

> A full description of the schema can be found on this [website][bidsschematools_rtd] 
> where you will also find the documentation for the python package
> to interact with the schema, [bidsschematools][bidsschematools_pypi].

Reviewer 2: Sorry, I'm pretty much always going to nit-pick anchor text.

```suggestion
A full description of the schema can be found in the [documentation][bidsschematools_rtd]
for [bidsschematools][bidsschematools_pypi], the Python package developed to interact with
the schema that is bundled with the specification.
```

https://developers.google.com/search/docs/crawling-indexing/links-crawlable#write-good-anchor-text is my go-to reference.
